Cargo.toml

```diff
@@ -10,13 +10,21 @@ repository = "https://github.com/stephaneyfx/brightness.git"
 documentation = "https://docs.rs/brightness"
 keywords = ["brightness", "backlight"]
 
+[features]
+async = ["async-trait", "futures", "tokio", "tokio-stream"]
```

Comment:
Makes the async features optional, for users who have no need for async in their projects (such as myself)